vendor/k8s.io/kubernetes/pkg/quota/v1/resources.go

```diff
@@ -0,0 +1,244 @@
+/*
```

Comment:
I don't see any reasons we would not want to have that on our end and not an upstream, but I might be missing some greater picture you have.

Reply:
> I don't see any reasons we would not want to have that on our end and not an upstream, but I might be missing some greater picture you have.

it comes in with the rebase.  This just makes workable early

Reply:
Can you add that to the list of followups so we don't forget about it?